app/src/main/java/woowacourse/movie/receiver/NotificationHelper.kt

```diff
@@ -0,0 +1,56 @@
+package woowacourse.movie.receiver
+
+import android.app.Notification
+import android.app.Notification.Builder
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.os.Build
+import androidx.core.app.NotificationCompat
+
+class NotificationHelper(
+    val context: Context, val channelId: String,
+    val channelName: String
```

Comment:
```suggestion
    val context: Context, 
    val channelId: String,
    val channelName: String
```
가급적이면 콤마 단위로 개행하는 것을 권장해요